feg/gateway/services/s8_proxy/servicers/mock_feg_relay/mock_feg_relay.go

```diff
@@ -20,29 +20,37 @@ import (
 
 	"magma/feg/cloud/go/feg"
 	"magma/feg/cloud/go/protos"
+	orc8r_protos "magma/orc8r/lib/go/protos"
+
 	"magma/feg/cloud/go/services/feg_relay"
 	"magma/orc8r/cloud/go/test_utils"
 )
 
 type TestS8ProxyResponderServer struct {
 	protos.S8ProxyResponderServer
 	ReceivedCreateBearerRequest *protos.CreateBearerRequestPgw
-	DefaultCreateBearerRes      *protos.CreateBearerResponsePgw
 	ListAddr                    string
+	Ready                       chan struct{}
 }
 
 func NewTestS8ProxyResponderServer() *TestS8ProxyResponderServer {
-	return &TestS8ProxyResponderServer{}
+	return &TestS8ProxyResponderServer{
+		Ready: make(chan struct{}),
```

Comment:
Whats the difference between `make(chan struct{})` and `chan struct{}` in this case?